src/_guides/platforms.md

```diff
@@ -25,16 +25,16 @@ or server, there's a Dart solution for that.
 
   Write an app that runs in any modern web browser.
 
-- {:.card.narrow}[![Dart logo][]{: width="100"}][Dart VM]{:.no-automatic-external}
+- {:.card.narrow}[![Dart logo][]{: width="100"}][Server]{:.no-automatic-external}
   {:.logo}
 
-  ### [Dart VM][]
+  ### [Server][]
 
-  Write a command-line app or server.
+  Write a command-line app or server for the Dart VM.
```

Comment:
Two comments about this sentence:

1. For the first part, how about:

    > Write a command-line or server-side app.

2. I'd be inclined to leave this sentence w/o the addition of "for the Dart VM".

    I _think_ that your intent was to somehow mention "Dart VM", but I'm unsure about the use of "for". A variant might be "... or server that runs on the Dart VM", but I still end up with the feeling that mentioning the Dart VM isn't that relevant.

@kwalrath WDYT?